rosbag2_py/src/rosbag2_py/format_bag_metadata.hpp

```diff
@@ -26,37 +27,17 @@
 #include <time.h>
 #endif
 
-namespace rosbag2_transport
-{
+#include "rosbag2_storage/bag_metadata.hpp"
 
-void Formatter::format_bag_meta_data(const rosbag2_storage::BagMetadata & metadata)
+namespace details
```

Comment:
This would be cleaner with a declaration of `std::string format_bag_meta_data` in a header, and all the implementation details in an unnamed namespace (`namespace {}`) to actually hide the symbols away. I'm not sure I like `namespace details` as a convention.

Reply:
@emersonknapp does ea4147330ff981598ac4f2a1fd851ef479d614b3 look better?

Reply:
yeah, much better